ortools/linear_solver/linear_solver_swig_helper.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2010-2021 Google LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// See go/mpsolver-callbacks for documentation on how to use this file.
```

Comment:
go/mpsolver-callbacks Where did you got this ?

Reply:
Hi!

I've copied it from here: https://github.com/google/or-tools/blob/stable/ortools/linear_solver/linear_solver_callback.h

I'll fix this and other issues tonight

Reply:
Remove this confusing string either for this file and linear_solver_callback.h